examples/conversation_with_RAG_agents/README.md

```diff
@@ -0,0 +1,40 @@
+# AgentScope Consultants: a Multi-Agent RAG Application
+
+* **What is this example about?**
+With the provided implementation and configuration,
+you will obtain three different agents who can help you answer different questions about AgentScope.
+
+* **What is this example for?** By this example, we want to show how the agent with retrieval augmented generation (RAG)
+capability can be used to build easily.
+
+## Prerequisites
+* **Cloning repo:** This example requires cloning the whole AgentScope repo to local.
+* **Packages:** This example is built on the LlamaIndex package. Thus, some packages need to be installed before running the example.
+    ```bash
+    pip install llama-index tree_sitter tree-sitter-languages
+    ```
+* **Model APIs:** This example uses Dashscope APIs. Thus, we also need an API key for DashScope.
+  ```bash
+  export DASH_SCOPE_API='YOUR_API_KEY'
+  ```
+  However, you are welcome to replace the Dashscope language and embedding models with other models you like.
+
```

Comment:
Please add a "##Note" section to remind users which model wrappers are tested already in this example.

Reply:
updated